data/scripts/sol-oic-gen.py

```diff
@@ -1512,10 +1529,12 @@ def seems_schema(path):
             continue
 
     print('\nWriting master JSON: %s' % args.node_type_json)
-    open(args.node_type_json, 'w+').write(master_json_as_string(generated))
+    open(args.node_type_json, 'w+').write(master_json_as_string(generated,
```

Comment:
@anselmolsm spotted that on systems without locale set to some `UTF-8`, it would fail to open files with non-ASCII symbols. Then please take the time to specify the encoding so we're safe.